src/cmd/cmdhelp.c

```diff
@@ -231,6 +231,8 @@ void emit_command_help (const char *plist, FILE *fp)
         emit_command_help_from_pattern (entry, fp);
 
     free (argz);
+
+    fprintf (fp, "\nSee also: https://flux-framework.readthedocs.io\n");
```

Comment:
This will require the `'flux-help command list can be extended'` test in `t0001-basic.t` to be modified, since unfortunately the test compares verbatim output from `flux help`.